db/migrate/20230214192802_add_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddNameToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :name, :string
+    add_column :users, :surname, :string
```

Comment:
- [ ] Kindly, as mentioned in the project requirement, only use the user's name.